website/src/components/pageStateSelectors/VariantsSelector.tsx

```diff
@@ -0,0 +1,92 @@
+    };
+
+    const addVariant = () => {
+        const newVariantFilterConfigs = new Map(variantFilterConfigs);
+
+        const newId = variantFilterConfigs.size === 0 ? 0 : Math.max(...Array.from(variantFilterConfigs.keys())) + 1;
+
+        newVariantFilterConfigs.set(newId, copyVariantFilterConfig(emptyVariantFilterConfig));
+
+        setVariantFilterConfigs(newVariantFilterConfigs);
+    };
+
+    const updateLineageFilterConfig = (id: Id, index: number, lineage: string | undefined) => {
+        const newVariantFilterConfigs = new Map(variantFilterConfigs);
+
+        const variantFilterConfig = newVariantFilterConfigs.get(id)!;
+
+        variantFilterConfig.lineageFilterConfigs[index].initialValue = lineage;
+
+        newVariantFilterConfigs.set(id, variantFilterConfig);
+
+        setVariantFilterConfigs(newVariantFilterConfigs);
+    };
+
+    const updateMutationFilter = (id: Id, mutation: MutationFilter | undefined) => {
+        const newVariantFilterConfigs = new Map(variantFilterConfigs);
+
+        const variantFilterConfig = newVariantFilterConfigs.get(id)!;
+
+        variantFilterConfig.mutationFilterConfig = mutation ?? {
+            nucleotideMutations: [],
+            aminoAcidMutations: [],
+            nucleotideInsertions: [],
+            aminoAcidInsertions: [],
+        };
+
+        newVariantFilterConfigs.set(id, variantFilterConfig);
+
+        setVariantFilterConfigs(newVariantFilterConfigs);
+    };
+
+    return (
+        <div>
+            <SelectorHeadline>Variant Filter</SelectorHeadline>
+            <div className='flex flex-col gap-4'>
+                {Array.from(variantFilterConfigs).map(([id, filterConfig]) => (
+                    <div key={id}>
+                        <div className='flex items-center justify-end'>
+                            <button className='btn btn-ghost btn-sm font-normal' onClick={() => removeVariant(id)}>
+                                Remove
+                            </button>
+                        </div>
+                        <div className='flex flex-col gap-2'>
+                            {filterConfig.lineageFilterConfigs.map((lineageFilterConfig, index) => (
+                                <LineageFilterInput
+                                    lineageFilterConfig={lineageFilterConfig}
+                                    onLineageChange={(lineage) => updateLineageFilterConfig(id, index, lineage)}
+                                    key={lineageFilterConfig.lapisField}
+                                />
+                            ))}
+                            <GsMutationFilter
+                                initialValue={getMutationFilter(filterConfig.mutationFilterConfig)}
+                                onMutationChange={(mutation) => updateMutationFilter(id, mutation)}
+                            />
+                        </div>
+                    </div>
+                ))}
```

Comment:
I would add a similar description as in #350. Then we don't show anything before the user has entered something. I would leave it like this, because the compare-side-by-side page works similarly. I added a ticket https://github.com/GenSpectrum/dashboards/issues/376